packages/typescript/index.js

```diff
@@ -44,6 +42,7 @@ function typescript (options) {
 
 function pre (context) {
   const registeredTypes = context.fileType.all()
+
   if (!('application/x-typescript' in registeredTypes)) {
```

Comment:
Would remove this pre hook, as it is now in the default mime types and the 2.0 update is a breaking one, so you have to update